mmdet3d/datasets/pipelines/loading.py

```diff
@@ -518,7 +618,7 @@ def __init__(self,
                  with_seg=False,
                  with_bbox_depth=False,
                  poly2mask=True,
-                 seg_3d_dtype=np.int64,
+                 seg_3d_dtype='int',
```

Comment:
Why do we need to modify this default setting?

Reply:
Sorry for the fault. I seem to commit the code based on my local version of mmdet3d last time and I have rectified that as the default setting.